common/models/templates/src/common.ts

```diff
@@ -1,148 +1,153 @@
-namespace models {
-  export const SENTINEL_CODE_UNIT = '\uFDD0';
-
-  export function applyTransform(transform: Transform, context: Context): Context {
-    // First, get the current context
-    let fullLeftContext = context.left || '';
-    let lLen = fullLeftContext.kmwLength();
-    let lDel = lLen < transform.deleteLeft ? lLen : transform.deleteLeft;
-
-    let leftContext = fullLeftContext.kmwSubstr(0, lLen - lDel) + (transform.insert || '');
-
-    let fullRightContext = context.right || '';
-    let rLen = fullRightContext.kmwLength();
-    let rDel = rLen < transform.deleteRight ? rLen : transform.deleteRight;
-
-    let rightContext = fullRightContext.kmwSubstr(rDel);
-
-    return {
-      left: leftContext,
-      right: rightContext,
-      startOfBuffer: context.startOfBuffer,
-      endOfBuffer: context.endOfBuffer,
-      casingForm: context.casingForm
-    };
-  }
+/// <reference types="@keymanapp/models-types" />
```

Comment:
I thought modules allowed us to eliminate `/// <reference`s?

The [docs](https://www.typescriptlang.org/docs/handbook/triple-slash-directives.html#-reference-types-) say:

> Use these directives only when you’re authoring a `d.ts` file by hand.

Reply:
... maybe this was also related to the sudden compiler strictness I'd experienced?  I seem to be able to remove it just fine now, at least.